src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -46,19 +51,52 @@
+           receiver-network-values       (if routes-available?
+                                           (send-utils/network-amounts to-network-values-for-ui
+                                                                       disabled-from-chain-ids
+                                                                       receiver-networks
+                                                                       true)
+                                           (send-utils/clean-network-amounts receiver-network-values))
+           network-links                 (when routes-available?
+                                           (send-utils/network-links chosen-route
+                                                                     sender-network-values
+                                                                     receiver-network-values))]
        {:db (-> db
                 (assoc-in [:wallet :ui :send :suggested-routes] suggested-routes-data)
                 (assoc-in [:wallet :ui :send :route] chosen-route)
                 (assoc-in [:wallet :ui :send :from-values-by-chain] from-network-values-for-ui)
                 (assoc-in [:wallet :ui :send :to-values-by-chain] to-network-values-for-ui)
+                (assoc-in [:wallet :ui :send :sender-network-values] sender-network-values)
+                (assoc-in [:wallet :ui :send :receiver-network-values] receiver-network-values)
+                (assoc-in [:wallet :ui :send :network-links] network-links)
                 (assoc-in [:wallet :ui :send :loading-suggested-routes?] false))}))))
 
 (rf/reg-event-fx :wallet/suggested-routes-error
- (fn [{:keys [db]} [_error]]
-   {:db (-> db
-            (update-in [:wallet :ui :send] dissoc :suggested-routes :route)
-            (assoc-in [:wallet :ui :send :loading-suggested-routes?] false))}))
+ (fn [{:keys [db]} [error]]
+   (let [receiver-network-values         (get-in db [:wallet :ui :send :receiver-network-values])
+         sender-network-values           (get-in db [:wallet :ui :send :sender-network-values])
+         cleaned-sender-network-values   (send-utils/clean-network-amounts sender-network-values)
+         cleaned-receiver-network-values (send-utils/clean-network-amounts receiver-network-values)]
+     {:db (-> db
+              (update-in [:wallet :ui :send]
+                         dissoc
+                         :route)
+              (assoc-in [:wallet :ui :send :sender-network-values] cleaned-sender-network-values)
+              (assoc-in [:wallet :ui :send :receiver-network-values] cleaned-receiver-network-values)
+              (assoc-in [:wallet :ui :send :loading-suggested-routes?] false)
+              (assoc-in [:wallet :ui :send :suggested-routes] {:best []}))
+      :fx [[:dispatch
+            [:toasts/upsert
+             {:id   :send-transaction-error
+              :type :negative
+              :text (:message error)}]]]})))
```

Comment:
We treat endpoint error as if no routes were found so user can retry the request, but we also show a toast to display the error